myo_stream:
  ros__parameters:
    device_id: "EC:6D:69:B3:F8:73"
    sampling_rate: 200.0
    emg_channels: [emg_0, emg_1, emg_2, emg_3, emg_4, emg_5, emg_6, emg_7]
    imu_channels: [accel_x, accel_y, accel_z, gyro_x, gyro_y, gyro_z, orient_x, orient_y, orient_z, orient_w]
    uuid_control: "d5060401-a904-deb9-4748-2c7f4a124842"
    uuid_emg_streams: ["d5060105-a904-deb9-4748-2c7f4a124842","d5060405-a904-deb9-4748-2c7f4a124842"]
    uuid_imu: "d5060402-a904-deb9-4748-2c7f4a124842"
    enable_cmd: [1,3,2,1,1,0,0]

gesture_recorder:
  ros__parameters:
    default_seconds: 5
    default_cycles: 2
    default_window_size: 200
    default_raw_mode: false
    default_out_dir: "myo_data"
    default_countdown_sec: 3

myo_rf:
  ros__parameters:
    window_size: 20

myo_cnn:
  ros__parameters:
    window_size: 40
    use_imu: false
    model_filename: "gesture_cnn_lstm.h5"
    norm_filename: "cnn_norm_stats.npz"
    label_encoder_filename: "label_encoder.pkl"
